### PR TITLE
more default args and negative step handling

### PR DESCRIPTION
**Pull request by smichr**
Modifications to allow `frange` to return a decreasing sequence and to warn on 0 jump.

### CODE REVIEW

File: frange.py [hunk 1]
```diff
@@ -63,3 +71,5 @@ def frange(start, stop, jump, end=False, via_str=False):
 
     assert list(frange(2, 3, '1/6', end=True))[-1] == 3.0
     assert list(frange(0, 100, '1/3', end=True))[-1] == 100.0
+    assert list(frange(4, jump=.11))[-1] == 3.96
```
Comment (smichr) [inline frange.py]:
```suggestion
```

File: frange.py [hunk 2]
```diff
@@ -33,18 +33,26 @@ def frange(start, stop, jump, end=False, via_str=False):
     >>> assert list(frange(1, 100.0, 0.1, end=True, via_str=True))[-1] == 100.0
 
     """
+    if float(jump) == 0:
+        raise ValueError('frange() arg 3 must not be zero')
+    if stop is None:
+      start, stop = 0, start
```
Comment (smichr) [inline frange.py]:
```suggestion
```

**Comment by marcotama** [comment]
Apologies for the late review. Looks good to me, thanks for the PR :)

**Comment by marcotama** [comment]
This addresses #1

**Pull request merged**